src/services/errorLog.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Error Log
+ *
+ * @module errorLog
+ */
+
+import http from './http.js';
+
+const postErrorLog = message => {
+  let currentTime = new Date();
+  let data = {
+    LOG_MESSAGE: message,
+    LOG_TIME: currentTime,
+  };
+  return http.post('log/add', data);
+};
+
+const postErrorMessage = message => {
+  return http.post('log', message);
+};
+
+//Code modified from https://medium.com/creative-technology-concepts-code/detect-device-browser-and-version-using-javascript-8b511906745
+function matchItem(string, data) {
```

Comment:
can you use the React function declaration syntax here, as you did on lines 9 and 18?